.github/workflows/gcf.yml

```diff
@@ -18,6 +18,7 @@ on:
     secrets:
       SLACKBOT_ENV_VARS:
         required: true
+  workflow_dispatch:
```

Comment:
This should let me trigger future deploys automatically from other branches in GH actions.